index.html

```diff
@@ -191,5 +213,46 @@ <h3 class="lets">LETS CONNECT</h3>
             </div>
         </div>    
     </section>
+
+    <section class="contac">
+        <div class="fond">
+            <img src="images/Contactme.jpg" alt="">
+        </div>
+        <div>
+            <h1 class="contacme">Contac me</h1>
+            <p class="textt">If you have an application you are interested in developing, 
+                a feature that you need built or a project taht needs coding. i d love to help with it.
+            </p>
+        </div>
+        <div class="for">
+            <form action="https://formspree.io/f/xknegzyn" method="post">
+                <ul class="point11">
+                    <li>
+                        <label for="name"></label>
+                        <input type="text" id="name" maxlength="30" required placeholder="Lucca Micale">
+                    </li>
+                    <li>
+                        <label></label>
+                        <input type="email" id="mail" required placeholder="luccamicale2015@gmail.com">
+                    </li>
+                    <li>
+                        <label></label>
+                        <textarea name="user_massage" required maxlength="500" id="msg" cols="30" rows="10" placeholder="Write your message here"></textarea>
+                    </li>
+                    <li class="button">
+                        <button type="submit" id="sub">Get in touch</button>
+                    </li>
+                </ul>
+            </form>
+        </div>
+        <div class="icons">
+            <ul class="icons-list">
+                <li class="rectan"></li>
+                <li class="rectan1"></li>
+            </ul>
+        </div>
+    
+    </section>
```

Comment:
- [ ] You have used position absolute for this whole section and for every element inside of it, this is not the recommended way, so please update it and use flexbox or grid instead